### ISSUE TITLE
does it work for conv2d_transpose?

### ISSUE THREAD

**Issue by bailin7134** [aaron-xichen/pytorch-playground#28]
does it work for conv2d_transpose?
I notice in quant.py, it works for nn.Conv2d, nn.Linear, nn.BatchNorm1d, nn.BatchNorm2d, nn.AvgPool2d. If I add nn.Conv2d_transpose, will it work?

**Comment by aaron-xichen** [comment]
It will add quantization layer after all nn.Conv2d_transpose
